clusterpools/apply.sh

```diff
@@ -187,6 +189,72 @@ generate_clusterimageset() {
     fi
 }
 
+generate_installconfigsecret() {
```

Comment:
I'd love for this to fill in the base domain if I've already given `CLUSTERPOOL_AWS_BASE_DOMAIN` and fill in the name from `CLUSTERPOOL_INSTALL_CONFIG_SECRET_NAME`.

Reply:
@dhaiducek are you meaning the install-config entry for CLUSTERPOOL_AWS_BASE_DOMAIN?  If so - that value is actually auto-filled-in by hive at runtime, you're supposed to leave that empty!

Reply:
Well okay, then! So should we put a brief comment so people like me don't wonder?
```
metadata:
  name: 
baseDomain: # skip, hive will inject it from the ClusterPool
```
Or leave it off entirely???

Reply:
Aha - again I copied straight from ACM 😆

Reply:
I'm a bit concerned that if we add a comment there we might break hive's injection 😨.  Let me merge this and then try it with a comment later!